services/viva-ms/lambdas/submitApplication.js

```diff
@@ -14,8 +14,10 @@ import vivaAdapter from '../helpers/vivaAdapterRequestClient';
 
 const dynamoDbConverter = AWS.DynamoDB.Converter;
 class TraceException extends Error {
-  constructor(message, customData) {
+  constructor(message, requestId, customData) {
```

Comment:
Could this class be extracted to a separate file. Maybe the class can be used elseware?

Is it necessary to create a class or can the same functionality be implemented in a function-based thinking?

Thoughts?